src/stdio_mgr/stdio_mgr.py

```diff
@@ -38,12 +37,7 @@
     TextIOWrapper,
 )
 
-# AbstractContextManager was introduced in Python 3.6
-# and may be used with typing.ContextManager.
-try:
-    from contextlib import AbstractContextManager
-except ImportError:  # pragma: no cover
-    AbstractContextManager = object
+from .types import _MultiCloseContextManager, ReplaceSysIoContextManager
```

Comment:
Please use absolute imports.  Once I get v1.0 of my new `flake8-absolute-import` published to PyPI, I'll be adding it here. :-)

Reply:
No worries.  I thinking about making the library easy to vendor, but I am not strongly motivated about it.

Reply:
If you currently have specific plans to vendor it, then I'm ok with using relatives.

Otherwise, I'd prefer absolutes.

Reply:
No real plans.